harmonic/install_windows_src.md

```diff
@@ -0,0 +1,153 @@
+   The Community version is free for students, open-source developers, and personal
+   development. Check "Desktop development with C++" in the Workloads tab,
+   check "MFC and ATL support", and uncheck "C++ Cmake Tools." We will install
+   cmake via Conda.
+
+3. Open a Visual Studio Command Prompt (search for "x64 Native Tools Command Prompt
+   for VS 2019" in the Windows search field near the Windows button). Optionally,
+   right-click and pin to the task bar for quick access in the future.
+
+  If you did not add Conda to your `PATH` environment variable during Conda installation,
+  you may need to navigate to the location of `condabin` in order to use the `conda` command.
+  To find `condabin`, search for "Anaconda Prompt" in the Windows search field near the
+  Windows button, open it, run `where conda`, and look for a line containing the directory `condabin`.
+
+4. Navigate to your `condabin`, if necessary, and then create and activate a Conda environment:
+  ```bash
+  conda create -n gz-ws
+  conda activate gz-ws
+  ```
+
+  Once you have activate an environment, a prefix like `(gz-ws)` will be prepended to
+  your prompt, and you can use the `conda` command outside of the `condabin` directory.
+
+  You can use `conda info --envs` to see all of your environments.
+
+  To remove an environment, use `conda env remove --name <env_name>`.
+
+5. Install dependencies:
+
+  ```bash
+  conda install cmake git vcstool curl pkg-config ^
+  colcon-common-extensions eigen freeimage gdal gts ^
+  glib dlfcn-win32 ffmpeg ruby tinyxml2 tinyxml ^
+  protobuf urdfdom zeromq cppzmq ogre jsoncpp ^
+  libzip qt pybind11 --channel conda-forge
+  ```
+
+6. Navigate to where you would like to build the library, create and enter your workspace directory,
+   create the `src` directory which will contain the Gazebo source code.
+  ```bash
+  mkdir gz-ws
+  cd gz-ws
+  mkdir src
+  ```
+
+7. Then clone the repositories
+  ```bash
+    # CMD
+    curl -sk https://raw.githubusercontent.com/gazebo-tooling/gazebodistro/master/collection-harmonic.yaml -o collection-harmonic
+    vcs import src < collection-harmonic
+
+    # PowerShell
+    curl https://raw.githubusercontent.com/gazebo-tooling/gazebodistro/master/collection-harmonic.yaml -o collection-harmonic
+    vcs import --input collection-harmonic src
+  ```
+
+## Building the Gazebo Libraries
+
+Once the compiler and all the sources are in place it is time to compile them.
+Start the procedure by navigating to your workspace and listing the packages
+recognized by `colcon`:
+
+```bash
+colcon graph
+```
+
+`colcon graph` should list the Gazebo libraries with an
+[interdependency diagram](https://colcon.readthedocs.io/en/released/reference/verb/graph.html#example-output).
+If that is the case, then you are ready to build the whole set of libraries:
+
+```bash
+colcon build --cmake-args -DBUILD_TESTING=OFF --merge-install --packages-up-to gz-sim7
```

Comment:
According with the [collection-garden.yaml](https://raw.githubusercontent.com/ignition-tooling/gazebodistro/master/collection-harmonic.yaml) file Gazebo version is the one in `main` who corresponds with `gz-sim8`

Reply:
[5f8e3e2](https://github.com/gazebosim/docs/pull/324/commits/5f8e3e20eb115262e864f8d151a85f3cf7852096)